actionlist/__tests__/reducers/actionItemsSpec.js

```diff
@@ -212,3 +211,32 @@ describe('action_items', () => {
     expect(reducer(beforeState, action)).toEqual(afterState);
   });
 });
+
+describe('action_items', () => {
+  it('should have the colors reset to white on pressing reset button', () => {
+    const beforeState = {
+      dropdownColors: {
+        Saved: preferenceColors.aqua,
```

Comment:
@burnumd  **preferenceColors** are the set of colors used only in preferences page. This has been separated from **Colors**  since it has all the colors for entire application.